tests/lint/git-black.sh

```diff
@@ -54,12 +54,16 @@ if [ -z ${FILES+x} ]; then
 fi
 echo "Files: $FILES"
 
+# required to make black's dep click to work
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
```

Comment:
i think this should actually just go in all Dockerfiles, no? i think most terminals should set it, but because Docker runs things in isolation, it doesn't forward those from the terminal. In addition, Jenkins can often be configured to set these (and we should ensure our Jenkins is setting them like this). i was kinda surprised the cuda container doesn't set it, but i don't see such a command [here](https://hub.docker.com/layers/nvidia/cuda/11.0.3-cudnn8-devel-ubuntu18.04/images/sha256-17e7999ad243f819a4a796e17486a11cc43fc94d9904783c36bc9d3117cabaf9?context=explore).